packages/front-b2b-link-rn/src/Types.ts

```diff
@@ -0,0 +1,80 @@
+export type BrokerType =
```

Comment:
We'll have to keep this type consistent with one in front-web-platform as we add/modify it there, can we consider having it shared between repos or it's not a good idea ?

Reply:
First attempt was to include directly from [JS SDK](https://github.com/FrontFin/front-b2b-link/blob/main/packages/link/src/utils/types.ts), but @balint-front was rightfully concerned about bloating the project with dependencies. Maybe we should have a small public repo for resources shared between front-web-platform and the various SDKs?

Reply:
agree

Reply:
I think that should be the end goal. But I suggest to move it after the release.

Reply:
Yes, we can move it after the release.